corehq/apps/hqadmin/views.py

```diff
@@ -458,6 +460,43 @@ def page_context(self):
         return context
 
 
+class VCMMigrationView(BaseAdminSectionView):
+    urlname = 'vcm_migration'
+    page_title = ugettext_lazy("Vellum Case Management Migration")
+    template_name = 'hqadmin/vcm_migration.html'
+
+    @property
+    def page_context(self):
+        domain_audit = namedtuple('domain_audit', 'name emailed migrated notes')
+        context = get_hqadmin_base_context(self.request)
+        context.update({
+            'audits': VCMMigration.objects.order_by('-migrated', '-emailed', 'domain'),
+            'url': reverse(self.urlname),
+        })
+        return context
+
+    def post(self, request, *args, **kwargs):
+        action = self.request.POST['action']
+        if action == 'notes':
+            m = VCMMigration.objects.get(domain=self.request.POST['domain'])
+            m.notes = self.request.POST['notes']
+            m.save()
+        else:
+            domains = self.request.POST['domains'].split(",")
+            for d in domains:
+                m = VCMMigration.objects.get(domain=d)
+                if action == 'email':
+                    send_mail_async.delay(
+                        _(u'Upcoming migration to easy references.'),
```

Comment:
@krsdimagi Can you send me the actual subject line & email copy? And also confirm that this email should come from `commcarehq-support@dimagi.com`?